x-pack/test/cloud_security_posture_functional/pages/benchmark.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { k8sFindingsMock } from '../mocks/latest_findings_mock';
+import type { FtrProviderContext } from '../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const kibanaServer = getService('kibanaServer');
+  const pageObjects = getPageObjects([
+    'common',
+    'cspSecurity',
+    'cloudPostureDashboard',
+    'rule',
+    'header',
+    'benchmark',
+    'findings',
+  ]);
+
+  describe('Access with custom roles', async () => {
+    let cspSecurity = pageObjects.cspSecurity;
+    let rule: typeof pageObjects.rule;
+    let benchmark: typeof pageObjects.benchmark;
+    let findings: typeof pageObjects.findings;
+
+    before(async () => {
+      benchmark = pageObjects.benchmark;
+      cspSecurity = pageObjects.cspSecurity;
+      await benchmark.waitForPluginInitialized();
+      await kibanaServer.savedObjects.clean({
+        types: ['cloud-security-posture-settings'],
+      });
+
+      await cspSecurity.createRoles();
+      await cspSecurity.createUsers();
+    });
+    after(async () => {
+      await cspSecurity.cleanRoles();
+      await cspSecurity.cleanUsers();
+    });
+
+    // Blocked by https://github.com/elastic/kibana/issues/184621
+    it.skip('Access with valid user role', async () => {
+      await cspSecurity.logout();
+      await cspSecurity.login('csp_read_user');
+      await benchmark.navigateToBenchnmarkPage();
+      expect(await benchmark.benchmarkPage.doesBenchmarkTableExists());
+    });
+
+    // Blocked by https://github.com/elastic/kibana/issues/184621
+    it.skip('todo - Access with invalid user role', async () => {});
```

Comment:
both this and the other todos can be commented out

Reply:
It's not a duplicate. one for `valid` user and the other for `invalid` user.

Reply:
![image](https://github.com/elastic/kibana/assets/51442161/411d44cb-44dd-47e9-8f7e-506d3dac8bc6)

sorry if im missing something but they read the same, probably just a typo so its better to fix it

Reply:
The second one is for the rule page.
Please see the comment:
```
    // The entire describe block bellow should move to rule.ts byt the page test is blocked by:
    // FLAKY: https://github.com/elastic/kibana/issues/178413
```